app/Http/Controllers/AppController.php

```diff
@@ -112,13 +114,14 @@ public function get_notifications(Request $request){
             return response($error_response);
         }
 
-        $app_user = AppUser::find($request->user_id);
+        $id = AppUser::where('rollNo','=',$request->rollNo)->value('id');
+        $app_user = AppUser::find($id);
```

Comment:
Recommended : use `$app_user = AppUser::where('rollNo',$request->rollNo)->first();` and remove the previous statement.

Reply:
done